patches/components-policy-tools-generate_policy_source.py.patch

```diff
@@ -1,20 +1,12 @@
 diff --git a/components/policy/tools/generate_policy_source.py b/components/policy/tools/generate_policy_source.py
-index a17b30c37e07a880ac6b588e08377394956e9b3a..15645bb54333bea9b2d7274c38dc696e71ff795f 100755
+index a17b30c37e07a880ac6b588e08377394956e9b3a..c6719a9254457cb0b293eee4e1ab66aafd8c93b4 100755
 --- a/components/policy/tools/generate_policy_source.py
 +++ b/components/policy/tools/generate_policy_source.py
-@@ -43,6 +43,7 @@ PLATFORM_STRINGS = {
-     'chrome.win7': ['win'],
- }
+@@ -1851,5 +1851,7 @@ def _WriteAppRestrictions(policies, policy_atomic_groups, target_platform, f,
+   f.write('</restrictions>')
  
-+from policy_source_helper import AddBravePolicies, CHROMIUM_POLICY_KEY
  
- class PolicyDetails:
-   """Parses a policy template and caches all its details."""
-@@ -342,6 +343,7 @@ def main():
-     chrome_major_version = args.chrome_version_major
- 
-   template_file_contents = _LoadJSONFile(template_file_name)
-+  AddBravePolicies(template_file_contents)
-   risk_tags = RiskTags(template_file_contents)
-   policy_details = [
-       PolicyDetails(policy, chrome_major_version, deprecation_milestone_buffer,
++from import_inline import inline_file_from_src;
```

Comment:
pls do this in a single line:
`from import_inline import inline_file_from_src; inline_file_from_src("brave/chromium_src/components/policy/tools/generate_policy_source.py", globals(), locals())`

Reply:
fixed in fcae15fc2d2700dfb36e7ff13a91e1e864c28608